lib/rubocop.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'English'
+require 'fileutils'
```

Comment:
I think it'd be a good idea to explain the need for this require in a comment, as its not directly used.